controllers/auth.js

```diff
@@ -9,10 +9,25 @@ module.exports.login = function (req, res) {
     })
 };
 
-module.exports.register = function (req, res) {
-    let user = new User({
-        email: req.body.email,
-        password: req.body.password
-    });
-    user.save().then(() => console.log('user  created'))
-};
\ No newline at end of file
+
+module.exports.register = async function (req, res) {
+    let candidate = await User.findOne({email: req.body.email})
+    if(candidate) {
+        res.status(409).json({
+            message: "this email is exist"
+        })
+    } else {
+        const user = new User({
+          email: req.body.email,
+          password: req.body.password
+        })
+        try {
+          await user.save();
+          res.status(200).json(user);
```

Comment:
It's looking good. But you had to write status code 201 - "Create" instead of 200